book/UpperLimitsTable.ipynb

```diff
@@ -716,7 +567,7 @@
    "source": [
     "## Acknowledgements\n",
     "\n",
-    "We acknowledge the help of ATLAS Statistics Forum including (alphabetical order):\n",
+    "We acknowledge discussions with the ATLAS Statistics Forum and the help of following:\n",
```

Comment:
all people listed *are* part of ATLAS Stats Forum. They're not all on the committee, of course, but are part of the forum.

Reply:
Reverted. I'm going to ask a very dumb question, sorry, but what is the Forum then? If it is just the mailing list then I don't seem to understand why we need to mention it versus just the people who gave information. But I am now just asking about minutia that isn't helpful in this review so ignore this.

Reply:
It is the mailing list, but these sorts of things were decided on or formulated before my time, and that particular mailing list had the people who were around when this procedure was discussed...